src/StaticWebAssetsSdk/Targets/Microsoft.NET.Sdk.StaticWebAssets.JSModules.targets

```diff
@@ -98,13 +98,19 @@ Copyright (c) .NET Foundation. All rights reserved.
          to identify them and correctly clasify them. Modules from other projects or packages will already be correctly tagged when we
          retrieve them.
     -->
+
+    <PropertyGroup>
+      <_ResolveJsInitializerModuleStaticWebAssetsCachePath>$(_StaticWebAssetsManifestBase)rjimswa.dswa.cache.json</_ResolveJsInitializerModuleStaticWebAssetsCachePath>
+    </PropertyGroup>
+
     <DefineStaticWebAssets Condition="@(_JSModuleCandidates) != ''"
       CandidateAssets="@(_JSModuleCandidates)"
       AssetTraitName="JSModule"
       AssetTraitValue="JSLibraryModule"
       RelativePathFilter="**/$(PackageId).lib.module.js"
       PropertyOverrides="AssetTraitName;AssetTraitValue"
       AssetMergeSource="$(StaticWebAssetMergeTarget)"
+      CacheManifestPath="$(_ResolveJsInitializerModuleStaticWebAssetsCachePath)"
```

Comment:
What's the criteria you used when deciding whether to specify a `CacheManifestPath`?

Reply:
Mostly whether you are potentially processing lots of items. I think for the time being its defined for all of the calls to DefineStaticWebAssets except for the scoped CSS ones